fullnumlistdivisors/fullnumlistdivisors.py

```diff
@@ -0,0 +1,42 @@
+#!python3
+# script that prints first N numbers divisible by --divisors=3,5,77,...
+import argparse
+from typing import List, Dict, Set
+#from sortedcontainers import SortedSet
+
+parser = argparse.ArgumentParser(description="The script prints first N numbers divisible by 3 or 5")
+parser.add_argument("-n", required=True, type=int, help="integer to show the list of numbers divisible by 3 or 5 ")
+parser.add_argument("-d", "--divisors", required=True, help="")
+args = parser.parse_args()
+
+def unique_divisors_set(unique_set: Set[int], dividor: int) -> Set[int]:
+    unique_set.add(dividor)
+    return unique_set
```

Comment:
No need to have this function since it's just one line and can be done in place

Reply:
got it